realtime/utils.js

```diff
@@ -471,6 +461,5 @@ module.exports = {
   isIpWhitelisted,
   parseObject,
   shouldIEmitThisObj,
-  isThisSample,
```

Comment:
I think we do not need the `attachAspectSubject` function too.

Reply:
attachAspectSubject is used a lot in cache/model/samples

Reply:
It's used by redisPublisher.pubishSample

Reply:
Ok, got it.